src/pages/search/search.styles.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh - 200px);
```

Comment:
I don't think you need a fixed height here. In general, I'd refrain from using a fixed height or width if not necessary